numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
> where is this used or built?
I mean where is `libpcore` used or built.  I dont see it anywhere (like in a Makefile or in a build script)